src/xclim/indicators/atmos/_precip.py

```diff
@@ -815,3 +816,15 @@ class HrPrecip(Hourly):
     cell_methods="time: sum over days",
     compute=indices.water_cycle_intensity,
 )
+
+antecedent_precipitation_index = Indicator(
+    title="Antecedent Precipitation Index",
+    identifier="antecedent_precipitation_index",
+    realm="atmos",
+    units="mm",
+    long_name="Antecedent Precipitation Index",
+    description="The Antecedent Precipitation Index, defined as running weighted sum of daily precipitation totals.",
+    abstract="The running sum of weighted, daily precipitation.",
+    cell_methods="time: running weighted sum over days",
```

Comment:
```suggestion
    cell_methods="time:  sum over days",
```
I don't know that `running weighted` are acceptable CF keywords ? But I haven't found anything similar in the [conventions](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.12/cf-conventions.html) either.